Sources/BraintreeCore/ConfigurationCache.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+
+class ConfigurationCache {
+        
+    // MARK: - Private Properties
+    
+    private let timeToLiveMinutes = 5.0
```

Comment:
curious why we set this to 5 min 👀

Reply:
It's been set as 5 minutes for a while on Android, so we followed suit on iOS. We figured 5 minutes was a reasonable time from a merchant configuring an APIClient to actually using the APIClient in a feature client to check out. But this can be whatever we want as long as it's consistent across platforms.